### PR TITLE
[css-lists] Test counter-reset: reversed()

### PR DESCRIPTION
**Pull request by zcorpan**
See https://github.com/w3c/csswg-drafts/pull/6096

### CODE REVIEW

File: css/css-lists/counter-reset-reversed-list-item.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Lists: counter-reset reversed(list-item)</title>
+<link rel="help" href="https://drafts.csswg.org/css-lists/#counter-reset">
+<link rel="match" href="counter-7-ref.html">
+<style>
+  .li { display: list-item; }
+  .reset-reversed { counter-reset: reversed(list-item) }
```
Comment (tabatkins) [inline css/css-lists/counter-reset-reversed-list-item.html:8]: Looking at the spec again, the rule for calculating the default value here is wrong - it says:

> the user agent must dynamically calculate the initial value at layout-time to be the number of elements and pseudo-elements that instantiate the same counter in the same scope, plus 1.

But you only instantiate a counter when you're creating a new one. You want to instead count, I guess, the number of elements that affect the counter? Or perhaps pretend it starts at -1, accumulate all the increments it receives, then start it at the negation of the final result (so it'll end on 1)?

File: css/css-lists/counter-reset-reversed-not-list-item.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<meta charset="utf-8">
+<title>CSS Lists: counter-reset reversed(not-list-item)</title>
+<link rel="help" href="https://drafts.csswg.org/css-lists/#counter-reset">
+<link rel="match" href="counter-7-ref.html">
+<style>
+  span { counter-increment: not-list-item -2; }
+  .reset-reversed { counter-reset: reversed(not-list-item) }
+  .result::before { content: counter(not-list-item) }
+</style>
+<p>You should see the number 7 below.</p>
```
Comment (tabatkins) [inline css/css-lists/counter-reset-reversed-not-list-item.html:11]: It looks like you're expecting the reversed counter to start on 13 here, meaning you're leaning towards "count the number of elements that apply an increment of any kind to the counter" for the impl.

If we're already dynamically counting things that actively increment the counter, wouldn't we want to do it "correctly" and take the actual increment into account, like I suggest above?

**Review by tabatkins — CHANGES_REQUESTED** [review]

**Comment by zcorpan** [comment]
Thanks @tabatkins , I'll comment in https://github.com/w3c/csswg-drafts/issues/6233 since the spec is wrong.

**Comment by zcorpan** [comment]
These tests need to be updated to match https://github.com/w3c/csswg-drafts/pull/6297

File: css/css-lists/counter-reset-reversed-not-list-item.html (hunk 2 above)
Comment (zcorpan) [reply css/css-lists/counter-reset-reversed-not-list-item.html]: This is now fixed.

File: css/css-lists/counter-reset-reversed-list-item.html (hunk 1 above)
Comment (zcorpan) [reply css/css-lists/counter-reset-reversed-list-item.html:8]: Spec fixed in https://github.com/w3c/csswg-drafts/pull/6297

**Comment by zcorpan** [comment]
Done.

**Review by tabatkins — APPROVED** [review]

**Comment by tabatkins** [comment]
r+, but it looks like we've got transient network failures causing some of the checks to fail. I think you might need to do a contentless push to kick them again.

**Pull request merged**